servlet25/src/main/java/brave/webmvc/Backend.java

```diff
@@ -0,0 +1,25 @@
+package brave.webmvc;
+
+import java.util.Date;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class Backend {
+
+  @RequestMapping("/api")
+  public ResponseEntity<String> printDate(
+      @RequestHeader(value = "user-name", required = false) String username
```

Comment:
ex if extra properties work, when the header "user-name" is passed to the frontend, you can read it on the backend